src/components/Accordion/Accordion/__tests__/accordion.interactions.ts

```diff
@@ -1,8 +1,8 @@
 import { expect } from "@storybook/jest";
-import { userEvent, within } from "@storybook/testing-library";
+import { within } from "@storybook/testing-library";
 import { Screen } from "@testing-library/react";
-import { delay, interactionSuite, resetFocus } from "../../../../__tests__/interactions-helper";
-
+import { resetFocus } from "../../../../__tests__/interactions-helper";
+import { userEvent, delay, interactionSuite } from "../../../../../dist/storybook-utils/services/interactions.js";
```

Comment:
for using compiled version and show it's working. If you think it will be better I can try import it form source (even that the code there should not be compiled from webpack like other code)